recipes-core/systemd/systemd_%.bbappend

```diff
@@ -12,6 +12,12 @@ do_install_append() {
     	echo "Address=${STATIC_IP_ADDRESS}/24" >> ${D}${sysconfdir}/systemd/network/10-pelux-ethernet.network
     fi
     ln -s /run/systemd/resolve/resolv.conf ${D}${sysconfdir}/resolv.conf
+
+    # Create mutiple consoles
+    tmp="${TTY_LIST}"
+    for ttydev in $tmp ; do
```

Comment:
It should be possible to do just `for ttydev in "${TTY_LIST}"`, right?

Reply:
To make it look clean I have done this way, as in other .bb file systemd-serialgetty.bb  they have done this way.